spec/support/coverage.rb

```diff
@@ -0,0 +1,11 @@
+unless defined? RUBY_ENGINE and RUBY_ENGINE == 'jruby'
```

Comment:
It's not in a great state atm, no.  https://github.com/colszowka/simplecov/issues/86

Even though hub runs on JRuby in production, would it be OK to also test it against MRI, not least so that we can get *some* coverage metrics?